lib/puppet/type/krb5kdc_auto_keytabs.rb

```diff
@@ -146,8 +146,6 @@
       The realms under which the hosts should be generated
     EOM
 
-    defaultto(Facter.value(:domain))
```

Comment:
This should not be removed. This is the de-facto standard for the `keytab` realm in almost all situations.

Reply:
It's still set in the class where this type is called

Reply:
@jeefberkey It still needs to default properly in the type. Either that, or be a required parameter, but the de-facto default is still the domain.